server/config.js

```diff
@@ -34,7 +31,6 @@ const establishments = {
     prefix: 'HMP',
     uuId: 'fd1e1db7-d0be-424a-a3a6-3b0f49e33293',
     standFirst: 'What you need to do to get or change your job in Berwyn.',
```

Comment:
is `.standFirst` related to GAJ? could we remove?

Reply:
Good spot!!! Will remove :)

Reply:
Actually, it appears to be used in the flat_content template and for that it doesn't use a path from config so I'll just remove the getting a job bits :)

Reply:
Awesome 💯

Reply:
It's also in the basic page template.

Reply:
Done